runtime/lua/vim/_meta/options.lua

```diff
@@ -7759,6 +7759,15 @@ vim.o.winbl = vim.o.winblend
 vim.wo.winblend = vim.o.winblend
 vim.wo.winbl = vim.wo.winblend
 
+--- Configures the border style of floating windows, border values only
+--- support string type which defined in `nvim_open_win()`.
+---
+--- @type string
```

Comment:
if it only support certain values why not directly make this an enum?

Reply:
we will have a dict type of option value in future that support custom character .this is in plan.

Reply:
ig this is meant https://github.com/neovim/neovim/pull/31084